pkg/controller/migplan/validation.go

```diff
@@ -48,6 +48,7 @@ const (
 	NsNotFoundOnDestinationCluster             = "NamespaceNotFoundOnDestinationCluster"
 	NsLimitExceeded                            = "NamespaceLimitExceeded"
 	NsLengthExceeded                           = "NamespaceLengthExceeded"
+	NotDNSCompliantNamespece                   = "NotDNSCompliantNamespece"
```

Comment:
For consistency with the other errors, I'd use this instead:
```
         NsNotDNSCompliant     = "NamespaceNotDNSCompliant"
```

Reply:
If you can confirm that `NsLengthExceeded` and `LengthExceeded` aren't used anywhere else, you might also remove them since they're being replaced by the new conditions in this PR.

Reply:
They are getting used to checking source namespace, and the condition there is slightly different, so those variables are needed.

Reply:
Ahh ok, makes sense.